timvt/db/catalog.py

```diff
@@ -25,7 +25,23 @@
         jsonb_object(
             array_agg(column_name),
             array_agg(udt_name)
-        ) as coldict
+        ) as coldict,
+        (
+            SELECT
+                ARRAY[
+                    ST_XMin(extent.geom),
+                    ST_YMin(extent.geom),
+                    ST_XMax(extent.geom),
+                    ST_YMax(extent.geom)
+                ]
+            FROM (
+                SELECT
+                    coalesce(
+                        ST_Transform(ST_SetSRID(ST_EstimatedExtent(f_table_schema, f_table_name, f_geometry_column), srid), 4326),
+                        ST_MakeEnvelope(-180, -90, 180, 90, 4326)
+                    ) as geom
+            ) AS extent
+        ) AS bounds
```

Comment:
@bitner can you do a quick double check 🙏

Reply:
looks good to me. go go sql master vincent!

Reply:
LGTM!

Reply:
🥳  thanks @bitner 🙏